sdk/storage/azure-storage-file-share/tests/test_file_client.py

```diff
@@ -44,18 +43,20 @@ def _teardown(self, FILE_PATH):
                 pass
     # --Helpers-----------------------------------------------------------------
     def validate_standard_account_endpoints(self, service, service_type, protocol='https'):
-        self.assertIsNotNone(service)
-        self.assertEqual(service.account_name, self.account_name)
-        self.assertEqual(service.credential.account_name, self.account_name)
-        self.assertEqual(service.credential.account_key, self.account_key)
-        self.assertTrue(service.primary_endpoint.startswith('{}://{}.{}.core.windows.net/'.format(
-            protocol, self.account_name, service_type)))
-        self.assertTrue(service.secondary_endpoint.startswith('{}://{}-secondary.{}.core.windows.net/'.format(
-            protocol, self.account_name, service_type)))
+        assert service is not None
+        assert service.account_name == self.account_name
+        assert service.credential.account_name == self.account_name
+        assert service.credential.account_key == self.account_key
+        assert service.primary_endpoint.startswith('{}://{}.{}.core.windows.net/'.format(protocol, self.account_name, service_type)) is True
+        assert service.secondary_endpoint.startswith('{}://{}-secondary.{}.core.windows.net/'.format(protocol, self.account_name, service_type)) is True
 
     # --Direct Parameters Test Cases --------------------------------------------
     @FileSharePreparer()
-    def test_create_service_with_key(self, storage_account_name, storage_account_key):
+    @recorded_by_proxy
+    def test_create_service_with_key(self, **kwargs):
```

Comment:
Yeah, tests like this that don't make any service requests shouldn't need to be recorded. It would seem most of the tests in this file fall into this category.

Reply:
Gotcha. I think I had asked you about this during the archboard meeting, but I realize describing it as "no service requests" rather than "empty recordings" would have more accurately described the recordings. Will fix 😄